src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -82,6 +95,11 @@ public override int KeySize
             }
         }
 
+        /// <summary>
+        /// Returns a <see cref="SignatureProvider"/> instance that supports the algorithm.
+        /// </summary>
+        /// <param name="algorithm">the algorithm to use for verifying/signing.</param>
+        /// <param name="verifyOnly">This value has to be false if need create Signatures.</param>
```

Comment:
This description isn't clear to me. Please work with Brent to get a better description here.